includes/polling/applications/opensearch.inc.php

```diff
@@ -0,0 +1,204 @@
+    'ttl_uncom_size' => $data['ttl_uncom_size'],
+    'ttl_last_mod_age' => $data['ttl_last_mod_age'],
+    'ti_total' => $data['ti_total'],
+    'ti_time' => $data['ti_time'],
+    'ti_failed' => $data['ti_failed'],
+    'ti_del_total' => $data['ti_del_total'],
+    'ti_del_time' => $data['ti_del_time'],
+    'ti_noop_up_total' => $data['ti_noop_up_total'],
+    'ti_throttled_time' => $data['ti_throttled_time'],
+    'ti_throttled' => $data['ti_throttled'],
+    'ts_q_total' => $data['ts_q_total'],
+    'ts_q_time' => $data['ts_q_time'],
+    'ts_f_total' => $data['ts_f_total'],
+    'ts_f_time' => $data['ts_f_time'],
+    'ts_sc_total' => $data['ts_sc_total'],
+    'ts_sc_time' => $data['ts_sc_time'],
+    'ts_su_total' => $data['ts_su_total'],
+    'ts_su_time' => $data['ts_su_time'],
+    'tr_total' => $data['tr_total'],
+    'tr_time' => $data['tr_time'],
+    'tr_ext_total' => $data['tr_ext_total'],
+    'tr_ext_time' => $data['tr_ext_time'],
+    'tf_total' => $data['tf_total'],
+    'tf_periodic' => $data['tf_periodic'],
+    'tf_time' => $data['tf_time'],
+    'tqc_size' => $data['tqc_size'],
+    'tqc_total' => $data['tqc_total'],
+    'tqc_hit' => $data['tqc_hit'],
+    'tqc_miss' => $data['tqc_miss'],
+    'tqc_cache_size' => $data['tqc_cache_size'],
+    'tqc_cache_count' => $data['tqc_cache_count'],
+    'tqc_evictions' => $data['tqc_evictions'],
+    'tg_total' => $data['tg_total'],
+    'tg_time' => $data['tg_time'],
+    'tg_exists_total' => $data['tg_exists_total'],
+    'tg_exists_time' => $data['tg_exists_time'],
+    'tg_missing_total' => $data['tg_missing_total'],
+    'tg_missing_time' => $data['tg_missing_time'],
+    'tm_total' => $data['tm_total'],
+    'tm_time' => $data['tm_time'],
+    'tm_docs' => $data['tm_docs'],
+    'tm_size' => $data['tm_size'],
+    'tm_throttled_time' => $data['tm_throttled_time'],
+    'tm_throttled_size' => $data['tm_throttled_size'],
+    'tw_total' => $data['tw_total'],
+    'tw_time' => $data['tw_time'],
+    'tfd_size' => $data['tfd_size'],
+    'tfd_evictions' => $data['tfd_evictions'],
+    'tseg_count' => $data['tseg_count'],
+    'tseg_size' => $data['tseg_size'],
+    'tseg_terms_size' => $data['tseg_terms_size'],
+    'tseg_fields_size' => $data['tseg_fields_size'],
+    'tseg_tvector_size' => $data['tseg_tvector_size'],
+    'tseg_norms_size' => $data['tseg_norms_size'],
+    'tseg_points_size' => $data['tseg_points_size'],
+    'tseg_docval_size' => $data['tseg_docval_size'],
+    'tseg_indwrt_size' => $data['tseg_indwrt_size'],
+    'tseg_vermap_size' => $data['tseg_vermap_size'],
+    'tseg_fbs_size' => $data['tseg_fbs_size'],
+    'trc_size' => $data['trc_size'],
+    'trc_evictions' => $data['trc_evictions'],
+    'trc_hits' => $data['trc_hits'],
+    'trc_misses' => $data['trc_misses'],
+    'tst_size' => $data['tst_size'],
+    'tst_res_size' => $data['tst_res_size'],
+];
+
+$rrd_name = ['app', $name, $app->app_id];
+$rrd_def = RrdDefinition::make()
```

Comment:
This is a large number of datasets.  Is there any chance to have variability in the data given by the client?

Reply:
Nope. It is very standardized as far as the return goes for both Elasticsearch and Opensearch from everything I've seen.

But aye, it is a large amount of datasets. Also OMG was so annoying going through and figuring out in some cases what was a counter v. gauge thanks to in some cases ambiguous docs/names.

Reply:
Something on your mind in regards to that all though, in terms of it all being in one RRD?

Reply:
Just annoying if for some reason the data changes later because you cannot simply change the DS in an existing RRD.  A little extra thought now can save pain later ;)

Reply:
woot! all cleaned up now